src/test/java/org/jitsi/videobridge/cc/SeenFrameTest.java

```diff
@@ -0,0 +1,93 @@
+package org.jitsi.videobridge.cc;
+
+import org.easymock.*;
+import org.jitsi.impl.neomedia.rtp.*;
+import org.jitsi.service.neomedia.*;
+import org.junit.*;
+import org.junit.experimental.runners.*;
+import org.junit.runner.*;
+import org.junit.runners.*;
+
+import java.util.*;
+
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.isA;
+import static org.junit.Assert.*;
+
+/**
+ * Created by bbaldino on 8/9/17.
+ */
+@RunWith(Enclosed.class)
+public class SeenFrameTest
+    extends EasyMockSupport
+{
+    @RunWith(Parameterized.class)
+    public static class SeenFrameAcceptTest
+        extends EasyMockSupport
+    {
+        @Parameterized.Parameters
+        public static Collection<Object[]> data()
+        {
+            return Arrays.asList(new Object[][] {
+                //  frameMinSeen,   frameMaxSeen,   pktSeqNum,  isMostRecentFrame,  isAdaptive, expectedResult
+                // Current frame, non-adaptive stream, sequence number in range
+                {   5,              10,             7,          true,               false,      true },
+                // Current frame, non-adaptive stream, sequence number not in range
+                {   5,              10,             12,         true,               false,      false },
+                // Current frame, adaptive stream, sequence number in range
+                {   5,              10,             7,          true,               true,       true },
+                // Current frame, adaptive stream, sequence number not in range
+                {   5,              10,             12,         true,               true,       false },
+                // Old frame, non-adaptive stream, sequence number in range
+                {   5,              10,             7,          false,              false,      true },
+                // Old frame, non-adaptive stream, sequence number not in range
+                {   5,              10,             12,         false,              false,      false },
+                // Old frame, adaptive stream, sequence number in range
+                {   5,              10,             7,          false,              true,       false },
+                // Old frame, adaptive stream, sequence number not in range
+                {   5,              10,             12,         false,              true,       false },
```

Comment:
Would it be better to put these values in a variable and then re-use or is that too much? That might reduce the comments.